src/ds/d500/d500-auto-calibration.h

```diff
@@ -3,12 +3,14 @@
 
 #pragma once
 
-#include "auto-calibrated-device.h"
+#include <src/auto-calibrated-device.h>
+#include <src/ds/d500/d500-auto-calibration-handler.h>
```

Comment:
Can use forward declaration for `d500_auto_calibrated_handler`, include in h

Reply:
include is needed here, because of the members that are of type of enums that are defined in the included header (see members _mode, _state, _result)